src/main/java/com/cegeka/academy/service/challenge/ChallengeServiceImp.java

```diff
@@ -162,6 +182,14 @@ private boolean isValidChallenge(Challenge challenge){
 
     }
 
+    private boolean isNextDay(Date date){
+
```

Comment:
aici nu verifici daca este ziua urmatoarea, ci daca data este in viitor. Te rog sa gasesti alt nume, sau implementeaza ceea ce zice numele actual